static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.spec.tsx

```diff
@@ -1350,7 +1344,7 @@ describe('WidgetBuilder', function () {
       userEvent.click(screen.getByText('Add Query'));
       userEvent.click(screen.getByText('Add Overlay'));
 
-      await screen.findByText('Limit to 2 results');
+      expect(screen.getByText('Limit to 2 results')).toBeInTheDocument();
```

Comment:
are you sure about this? doesn't it take some time to update the UI?

Reply:
Maybe, but if this passes with and without user action above, then I would say it is not a very strong test and we should think of a better way to test it

Reply:
I was afraid that this was going to fail in CI, but let's see ... hopefully not :)